File: FoodBack/src/main/java/cn/edu/fzu/sm2025/wwc/foodback/entity/Application.java
package cn.edu.fzu.sm2025.wwc.foodback.entity;

import lombok.Data;
import java.sql.Time;
import java.sql.Timestamp;

@Data
public class Application {
    private Integer applicationId;
    private String userName;
    private String shopName;
    private String shopLocation;
    private String shopDescription;
    private String shopCategory;
    private String openTime;
    private String closeTime;
    private String status;
    private Timestamp createdAt;
    private Timestamp reviewedAt;
    private String rejectionReason;
    private String imagePath;
}

File: FoodPlatform/entry/src/main/ets/model/Application.ts
//分页响应类型
export interface PageResult<T> {
  total: number;
  list: T[]
}

//申请实体类型
export interface Application{
  applicationId: number;
  userName: string;
  shopName: string;
  shopLocation: string;
  shopDescription: string;
  shopCategory: string;
  openTime: string;
  closeTime: string;
  status: string;
  createdAt: string;
  reviewedAt: string;
  rejectionReason: string;
  imagePath: string
}

//网络错误类型
export interface HttpError{
  code: number;
  message: string
}
